src/snowflake/cli/api/project/util.py

```diff
@@ -4,8 +4,9 @@
 from typing import Optional
 
 IDENTIFIER = r'((?:"[^"]*(?:""[^"]*)*")|(?:[A-Za-z_][\w$]{0,254}))'
+IDENTIFIER_NO_LENGTH = r'((?:"[^"]*(?:""[^"]*)*")|(?:[A-Za-z_][\w$]*))'
 DB_SCHEMA_AND_NAME = f"{IDENTIFIER}[.]{IDENTIFIER}[.]{IDENTIFIER}"
-SCHEMA_AND_NAME = f"{IDENTIFIER}[.]{IDENTIFIER}"
+SCHEMA_AND_NAME = f"{IDENTIFIER}[.]{IDENTIFIER_NO_LENGTH}"
```

Comment:
Can you explain this change?

Reply:
That was a lost artifact of long battle with regexes. Long story.